### PR TITLE
Use CloudWatchLogs module in browser_events_controller

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
Use the [new CloudWatchLogs module](https://github.com/code-dot-org/code-dot-org/pull/62485) in the `browser_events_controller` to buffer event load.

## Testing story

Tested locally; verified that events show up in the development log stream after the buffer is flushed.

### CODE REVIEW

File: dashboard/app/controllers/browser_events_controller.rb [hunk 1]
```diff
@@ -23,17 +24,7 @@ def put_logs
...
-        log_group_name: LOG_GROUP_NAME,
-        log_stream_name: LOG_STREAM_NAME,
-        log_events: logs
-      }
-    )
-
-    if resp.rejected_log_events_info
-      fallback_log_to_firehose('rejected-log-events', resp.rejected_log_events_info)
-    end
+    Cdo::CloudWatchLogs.put_log_events(LOG_GROUP_NAME, LOG_STREAM_NAME, logs)
```
Comment (sureshc) [inline dashboard/app/controllers/browser_events_controller.rb:25]: Can we remove the declaration of `LOGS_CLIENT` above?

``` ruby
LOGS_CLIENT = Aws::CloudWatchLogs::Client.new
```

File: dashboard/app/controllers/browser_events_controller.rb [hunk 2]
```diff
@@ -23,17 +24,7 @@ def put_logs
...
-    resp = LOGS_CLIENT.put_log_events(
-      {
-        log_group_name: LOG_GROUP_NAME,
-        log_stream_name: LOG_STREAM_NAME,
-        log_events: logs
-      }
-    )
-
-    if resp.rejected_log_events_info
-      fallback_log_to_firehose('rejected-log-events', resp.rejected_log_events_info)
```
Comment (sureshc) [inline dashboard/app/controllers/browser_events_controller.rb:35]: We could delete the declaration of the `fallback_log_to_firehose` method.

File: dashboard/app/controllers/browser_events_controller.rb (hunk 1 above)
Comment (sanchitmalhotra126) [reply dashboard/app/controllers/browser_events_controller.rb:25]: Good catch, done.

File: dashboard/app/controllers/browser_events_controller.rb (hunk 2 above)
Comment (sanchitmalhotra126) [reply dashboard/app/controllers/browser_events_controller.rb:35]: Done

**Comment by sureshc** [comment]
I didn't investigate why UI tests failed, but kicked off another continuous integration build in case with was a reliability issue with the test.

**Review by sureshc — APPROVED** [review]

**Comment by cat5inthecradle** [comment]
How confident are we of the logs buffer in prod? I was on vacation and sick when this got wrapped up, so I haven't had much firsthand experience with it yet (trying to gauge risk, and whether this should be a more isolated DTP during off hours)

**Pull request merged**